src/main/java/edu/ucsb/cs156/organic/controllers/UserInfoController.java

```diff
@@ -60,4 +62,12 @@ public Iterable<UserEmail> getUsersEmails() {
     return user.getEmails();
   }
 
+  @Operation(summary = "Get courses for which current user is on the staff")
+  @PreAuthorize("hasRole('ROLE_USER')")
+  @GetMapping("/staffedCourses")
```

Comment:
Im not sure this path makes sense given the operation description

Reply:
What name would you suggest instead?

I put this out to the staff meeting and asked what a better name would be and no one had any suggestion other than: "what does Guy think the name should be?" lol